modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -329,6 +330,66 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_timeline(array &$variables)
   $variables['hide_from'] = $paragraph->get('field_oe_timeline_expand')->value;
 }
 
+/**
+ * Implements hook_preprocess_paragraph() for paragraph--oe-content-row--variant-inpage-navigation.html.twig.
+ */
+function oe_bootstrap_theme_preprocess_paragraph__oe_content_row__variant_inpage_navigation(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+
+  if (!$paragraph->get('field_oe_title')->isEmpty()) {
+    $variables['title'] = $paragraph->get('field_oe_title')->value;
+  }
+
+  $variables['full_layout'] = TRUE;
+  $variables['scrollspy_id'] = 'bcl-inpage-navigation';
+
+  $links = [];
+  $content = [];
+
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $delta => $sub_paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+    if ($sub_paragraph->hasField('field_oe_title') && !$sub_paragraph->get('field_oe_title')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_title')->first()->value;
+    }
+    elseif ($sub_paragraph->hasField('field_oe_text') && !$sub_paragraph->get('field_oe_text')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_text')->first()->value;
+    }
+    elseif ($sub_paragraph->hasField('field_oe_plain_text_long') && !$sub_paragraph->get('field_oe_plain_text_long')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_plain_text_long')->first()->value;
```

Comment:
this one looks scary, how long can this be ?

Reply:
As long as it may be: we need to check the paragraph type Accordion. If we place an accordion in the inpage navigation we need to mount also the inpage stuff. Please, check it on a live case to understand what problems we have here. Then we can discuss the solutions (E.g: a) Add a title to an Accordion paragraph, b) Add code to avoid Accordion to be included in an inpage navigation....).